k8s/namespaces/nfdiv/nfdiv-cron-migrate-bulk-cases/perftest-00.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: nfdiv-cron-migrate-bulk-cases
+spec:
+  releaseName: nfdiv-cron-migrate-bulk-cases
+  values:
+    job:
+      suspend: true
+    global:
+      jobKind: CronJob
+      enableKeyVaults: true
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      environment: perftest
+      schedule: 0 0 * * *
```

Comment:
according to docs schedule is not a global value
https://github.com/hmcts/chart-job#configuration

Reply:
@timja Interesting, we have it configured in this line: https://github.com/hmcts/cnp-flux-config/blob/7a691baf54b0824bb15a71eeca07148f61989b6e/k8s/namespaces/nfdiv/nfdiv-cron-migrate-cases/perftest-00.yaml#L29